samples/angular/src/app/pillarbox.component.ts

```diff
@@ -13,9 +13,9 @@ export class PillarboxComponent implements AfterViewInit, OnChanges {
   @Input() type: string = 'srgssr/urn';
   @Input() src!: string;
 
-  private player?: PillarboxPlayer;
+  private player?: any;
   ngAfterViewInit(): void {
-    this.player = pillarbox('main-player') as PillarboxPlayer;
+    this.player = pillarbox('main-player');
```

Comment:
to help me understand, could you tell me what motivated this change?

Reply:
Stackblitz was not recognising the transitive video.js "Player" type correctly. I've explicitly added the dependency in the package.json so that it now it is installed in the Stackblitz workspace by default.